raiden/tasks.py

```diff
@@ -92,6 +93,20 @@ def check_gas_reserve(raiden):
         gevent.sleep(CHECK_GAS_RESERVE_INTERVAL)
 
 
+def check_network_id(network_id, web3: Web3):
```

Comment:
I thought about combining this and the gas reserve check -- but at least for now thought I can keep it flexible. Open to comments and suggestions.